src/lib/api/group/client/groupApiClient.client.ts

```diff
@@ -122,7 +122,7 @@ export const deleteMember = async (
 ): Promise<Result<undefined, string>> => {
   try {
     await request({
-      url: `${BACKEND_API_DOMAIN}/groups/${groupId}/users/${userId}`,
+      url: `${API_BASE_URL}/groups/${groupId}/users/${userId}`,
```

Comment:
@kanta1207  Sorry, I should've mentioned that, but you are right, I have changed it in order to avoid error.
I guess it happend because Nick was working on changing domain variable(in PR #267) at the time I was implementing delete member function( #268 )
Thanks for notice!